src/pymordemos/parabolic_mor.py

```diff
@@ -66,7 +66,7 @@ def main(
         rom, fom=fom, reductor=reductor, error_estimator=True,
         error_norms=[lambda U: DT * np.sqrt(np.sum(fom.h1_0_semi_norm(U)[1:]**2))],
         error_norm_names=['l^2-h^1'],
-        condition=False, test_mus=parameter_space.sample_randomly(test, seed=999)
+        condition=False, test_mus=parameter_space.sample_randomly(test)
```

Comment:
Just out of couriosity: donen't this mean this demo is executed with a different seed from now on?

Reply:
Yes, although random state would probably be the better word. In fact, the `test_mus` will depend on the basis generation method as `adaptive_greedy` draws additional random parameters, which the other methods do not do. I'm not sure if we want determinism here or not.

Reply:
Me neither, and probably doesn't matter. Was just asking to make sure I understand the changes :/